CMakeLists.txt

```diff
@@ -67,8 +67,8 @@ option(BUILD_QTSLIM "Build the Qt5 GUI for SLiM (on Linux only)" OFF)
 
 
 # Use the flags below for [all / Debug / Release] builds; these flags are built in to cmake
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11 -Wno-deprecated-register -Wno-attributes -Wunused-label -Wimplicit -Wunused-variable -Wunused-value -Wno-pragmas -Wimplicit-fallthrough -Wempty-body -Wshadow -Wparentheses -Wdangling-else -Wmissing-prototypes -Wswitch -Wpointer-sign -Wsign-compare -Wstrict-prototypes -Wconversion -Wno-sign-conversion -Wuninitialized")
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wno-deprecated-register -Wno-attributes -Wunused-label -Wimplicit -Wunused-variable -Wunused-value -Wno-pragmas -Wimplicit-fallthrough -Wempty-body -Wshadow -Wparentheses -Wdangling-else -Wmissing-prototypes -Wswitch -Wpointer-sign -Wsign-compare -Wstrict-prototypes -Wconversion -Wno-sign-conversion -Wuninitialized")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11 -Wno-attributes -Wunused-label -Wimplicit -Wunused-variable -Wunused-value -Wno-pragmas -Wimplicit-fallthrough -Wempty-body -Wshadow -Wparentheses -Wdangling-else -Wmissing-prototypes -Wswitch -Wpointer-sign -Wsign-compare -Wstrict-prototypes -Wno-sign-conversion -Wuninitialized")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wno-attributes -Wunused-label -Wunused-variable -Wunused-value -Wno-pragmas -Wimplicit-fallthrough -Wempty-body -Wshadow -Wparentheses -Wdangling-else -Wswitch -Wsign-compare -Wno-sign-conversion -Wuninitialized")
```

Comment:
I'm about to commit these same changes on the master branch; I'm trying to build on Ubuntu now and the warnings are bugging me.  So you can just revert this file back to the state of master.